hack/e2e-common.sh

```diff
@@ -57,3 +57,8 @@ function cluster_kueue_deploy {
     fi
 }
 
+export INITIAL_IMAGE=$(cd config/components/manager && $KUSTOMIZE cfg grep "kind=Kustomization" . | yq '.images[] | select(.name == "controller") | [.newName, .newTag] | join(":")')
```

Comment:
I don't love this complexity, but we can keep something along these lines.
```suggestion
export INITIAL_IMAGE=$(yq '.images[] | select(.name == "controller") | [.newName, .newTag] | join(":")' config/components/manager/kustomization.yaml)
```

Reply:
I like the `cd config/components/manager && $KUSTOMIZE cfg grep "kind=Kustomization" .` for symmetry , get and set with kustomize, however Done